Prace_domowe/Praca_domowa4/Jakub_Foltyn/PD4.ipynb

```diff
@@ -0,0 +1 @@
UWYSWpIk7ZbkAODngPMAqurfquoe4BTgglbtAuA5bfgU4MLquQo4MMmhK9xsSZIkSdIIWzPsBkiSpJFyJPBN4F1JngRcA7wMmKqqHa3O7cBUGz4MuK3v89ta2Y6+MpKcSe9Kaaamppienl52Q3fu3DmQ6YyySY9x0uMDY5wUxihJkrQ8JqElSVK/NcCxwEur6uokb+H+rjcAqKpKUouZaFWdC5wLsH79+tqwYcOyGzo9Pc0gpjPKJj3GSY8PjHFSGKMkSdLymITWyFi7+bJlT2Pr2ScPoCWStKptA7ZV1dXt/cX0ktB3JDm0qna07jbubOO3A0f0ff7wVqYhGcT2FNymSpIkLdb12+/l9GXui7kPpklln9CSJGm3qroduC3JE1rRCcCNwKXAxla2EbikDV8KvCg9xwP39nXbIUmSJEmSV0JLkqQHeSnw3iT7ArcAL6Z34vqDSc4AbgWe3+peDpwEbAG+2+pKkiRJkrSbSWhJkvQAVXUtsH6OUSfMUbeAszpvlCRJkiRpbNkdhyRJkiRJkiSpMyahJUmSJEmSJEmdMQktSZIkSZIkSeqMSWhJkiRJkiRJUmdMQkuSJEmSJEmSOmMSWpIkSZIkSZLUGZPQkiRJkiRJkqTOmISWJEmSJEmSJHXGJLQkSZIkSZIkqTMmoSVJkiRJkiRJnTEJLUmSJEmSJEnqjEloSZIkSZIkSVJnTEJLkiRJkjQBkhyR5NNJbkzy1SQva+UHJ7kiyc3t70GtPEnemmRLkuuSHNs3rY2t/s1JNg4rJknSZDAJLUmSJEnSZNgFbKqqo4HjgbOSHA1sBq6sqqOAK9t7gGcDR7XXmcA7oJe0Bl4NPAU4Dnj1TOJakqSlWFYSOsn/1c6u3pDk/UkemuTIJFe3M6kfSLJvq7tfe7+ljV87iAAkSZIkSRJU1Y6q+lIbvg+4CTgMOAW4oFW7AHhOGz4FuLB6rgIOTHIo8Czgiqq6q6ruBq4ATlzBUCRJE2bNUj+Y5DDgvwBHV9W/JPkgcCpwEvCmqrooyZ8DZ9A7m3oGcHdVPT7JqcAbgF9fdgSSJEmSJOkB2oVfTwauBqaqakcbdTsw1YYPA27r+9i2VjZf+ezvOJPeFdRMTU0xPT297Hbv3LlzINMZdaslTlhdsU49DDat27WsaYzDvFpNy9RYB2fJSei+zz8syfeBhwM7gKcDv9HGXwC8hl4S+pQ2DHAx8LYkqapaZhskSZIkSVKT5JHAh4GXV9V3kuweV1WVZCDH4VV1LnAuwPr162vDhg3Lnub09DSDmM6oWy1xwuqK9c/eewnnXL+8VNvW0zYMpjEdWk3L1FgHZ8ndcVTVduD/Bf6JXvL5XuAa4J6qmjnt03+2dPeZ1Db+XuDRS/1+SZIkSZL0QEkeQi8B/d6q+kgrvqN1s0H7e2cr3w4c0ffxw1vZfOWSJC3JcrrjOIje1c1HAvcAH2IAfUQt5Xae1XRpfJeGPR+Xe8sKjMZtK8Oej5PC+TgYzkdJkqTVI71Lns8DbqqqN/aNuhTYCJzd/l7SV/6SJBfRewjhvVW1I8kngP/W9zDCZwKvXIkYJEmTaTn3CPwC8I9V9U2AJB8BnkbvQQZr2tXO/WdLZ86kbkuyBjgA+PbsiS7ldp7VdGl8l4Y9H0/ffNmypzEKt60Mez5OCufjYDgfJUmSVpWnAS8Erk9ybSt7Fb3k8weTnAHcCjy/jbuc3nOdtgDfBV4MUFV3JflT4Aut3mur6q6VCUGSNImWk4T+J+D4JA8H/gU4Afgi8GngecBFPPgM60bgc238p+wPejKsHUDyWJIkPXibumndroGcpJUkrQ5V9Vkg84w+YY76BZw1z7TOB84fXOskSavZcvqEvpreAwa/BFzfpnUu8Arg95Nsodfn83ntI+cBj27lvw9sXka7JUmSJEmSJEljYFmP7KyqVwOvnlV8C3DcHHW/B/zacr5PkiRJkiRJkjRelnwltCRJkiRJkiRJe2MSWpIkSZIkSZLUGZPQkiRJkiRJkqTOmISWJEmSJEmSJHXGJLQkSXqAJFuTXJ/k2iRfbGUHJ7kiyc3t70GtPEnemmRLkuuSHDvc1kuSJEmSRo1JaEmSNJefr6pjqmp9e78ZuLKqjgKubO8Bng0c1V5nAu9Y8ZZKkiRJkkaaSWhJkrQQpwAXtOELgOf0lV9YPVcBByY5dBgNlCRJkiSNpjXDboAkSRo5BXwySQF/UVXnAlNVtaONvx2YasOHAbf1fXZbK9vRV0aSM+ldKc3U1BTT09PLbuTOnTsHMp1Rsmndrge8n3rYg8tWykrM20lchrMZ42QwRkmSpOUxCS1Jkmb72aranuSHgSuS/H3/yKqqlqBesJbIPhdg/fr1tWHDhmU3cnp6mkFMZ5ScvvmyB7zftG4X51w/nN21radt6Pw7JnEZzmaMk8EYJUmSlsfuOCRJ0gNU1fb2907go8BxwB0z3Wy0v3e26tuBI/o+fngrkyRJkiQJMAktSZL6JHlEkv1nhoFnAjcAlwIbW7WNwCVt+FLgRek5Hri3r9sOSZIkSZLsjkOSJD3AFPDRJNDbT3hfVX08yReADyY5A7gVeH6rfzlwErAF+C7w4pVvsiRJkiRplJmEliRJu1XVLcCT5ij/NnDCHOUFnLUCTZMkSZIkjSm745AkSZIkSZIkdcYktCRJkiRJkiSpMyahJUmSJEmSJEmdMQktSZIkSZIkSeqMSWhJkiRJkiRJUmdMQkuSJEmSJEmSOrNm2A3Q0qzdfNlAprP17JMHMh1JkiRJkiRJmotXQkuSJEmSJEmSOmMSWpIkSZIkSZLUGZPQkiRJkiRJkqTOmISWJEmSJEmSJHXGJLQkSZIkSZIkqTMmoSVJkiRJkiRJnTEJLUmSJEmSJEnqjEloSZIkSZIkSVLzCa0AAAAgAElEQVRnTEJLkiRJkiRJkjpjElqSJEmSJEmS1BmT0JIkSZIkSZKkzqwZdgM0XGs3X7Z7eNO6XZze916SJEmSJEmSlssroSVJ
```

Comment:
<p>Szkoda, że nie pokazano strojenia w przypadku skalowania - to był cel pracy domowej.</p><p>Należało dokonać porównać na dwóch poziomach:</p><ol><li>wynik modeli z i bez skalowania</li><li>wynik modeli z i bez skalowania uwzględniając strojenie</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>